create-container.sh

```diff
@@ -5,24 +5,32 @@ set -e
 # Uncomment the following line to debug the script
 # set -x
 
-# Load configuration
+function config_exists_or_exit {
+    CONFIG="$1"
+    if [ ! -f "$CONFIG" ]; then
+        echo "ERROR: needed config file \"$CONFIG\" does not exist."
+        exit 1
+    fi
+}
+
+PROJECT_CONFIG="$PWD/.devenv"
+config_file_exists_or_exit "$PROJECT_CONFIG"
+# Load project configuration
 # shellcheck source=/dev/null
-source "$PWD/.devenv"
+source "$PROJECT_CONFIG"
 
-# Defaults
-RETRIES=5
+# Load global defaults
+GLOBAL_CONFIG=/etc/devenv
+config_file_exists_or_exit "$GLOBAL_CONFIG"
+# shellcheck source=config
+source "$GLOBAL_CONFIG"
```

Comment:
Nice, maybe we can also output something after both project and global config sourcing, like:
```
Project configuration from /path/to/prjconf
Devenv configuration from /path/to/devenvconf
```
What do you think? It's just to make it easy to remind people where things come from.

Reply:
Totally!